### ISSUE TITLE
Question about the dataset

### ISSUE THREAD

**Issue by HUANGLIZI** [nikhilroxtomar/TGANet#3]
Could you provide the specific training set and testing set and validation set? please

**Comment by HUANGLIZI** [comment]
Could you provide the official split of 880/120 for training and testing on Kvasir-SEG dataset? At the same time, I noticed that the official split was not mentioned in the Kvasir-SEG dataset paper and website.

**Comment by nikhilroxtomar** [comment]
You can have a look at this repository: https://github.com/DebeshJha/ColonSegNet
It has both the train.txt and val.txt. Here the train.txt is used for training, while val.txt is used for both the validation and testing.
